financial-connections/src/main/java/com/stripe/android/financialconnections/domain/CreateInstantDebitsResult.kt

```diff
@@ -0,0 +1,48 @@
+package com.stripe.android.financialconnections.domain
+
+import com.stripe.android.financialconnections.launcher.InstantDebitsResult
+import com.stripe.android.financialconnections.repository.ConsumerSessionProvider
+import com.stripe.android.financialconnections.repository.FinancialConnectionsConsumerSessionRepository
+import com.stripe.android.financialconnections.repository.FinancialConnectionsRepository
+import com.stripe.android.model.ConsumerPaymentDetails.BankAccount
+import javax.inject.Inject
+
+internal fun interface CreateInstantDebitsResult {
+    suspend operator fun invoke(
+        bankAccountId: String,
+    ): InstantDebitsResult?
```

Comment:
Can this ever be null?

Reply:
It can’t. Updated the interface definition.